src/cloud-api-adaptor/libvirt/config_libvirt.sh

```diff
@@ -12,6 +12,7 @@ set -o pipefail
 source /etc/os-release || source /usr/lib/os-release
 ARCH=$(uname -m)
 TARGET_ARCH=${ARCH/x86_64/amd64}
+TARGET_ARCH=${ARCH/aarch64/arm64}
```

Comment:
This line is breaking the x86 as it overrides the previous line. I think this
```suggestion
TARGET_ARCH=${TARGET_ARCH/aarch64/arm64}
```
would work though, or something else more elegant?

Reply:
Thanks, will fix this shortly.

Reply:
How about follow the `OS_DISTRO`? easier to read and apparently it only supports ubuntu and rhel, similar with ARCH.
```suggestion
if [[ "$ARCH" == "aarch64" ]]; then
    TARGET_ARCH=arm64
fi
```

Reply:
Yeah, that approach is fine by me too